app/views/layouts/mailer.html.erb

```diff
@@ -1,30 +1,29 @@
 <html>
-<body style='font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, sans-serif;'>
-<table style="width:100%;max-width:700px;margin:auto;font-size:18px">
-  <tr>
-    <td style="padding:4%;">
-      <%= yield %>
-    </td>
-  </tr>
-  <% if @user %>
-    <tr>
-      <td style="padding:3% 12% 2% 4%;font-size:18px;line-height:22px;color:#7d7d7d">
-        <% providers = authentication_enabled_providers_for_user(@user).map(&:official_name) %>
-        Reminder: You used <b><%= providers.to_sentence %></b> to authenticate your account, so use <%= providers.size == 1 ? "that" : "any of those" %> to sign in if prompted.
+  <body style='font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, sans-serif;'>
+    <table style="width:100%;max-width:700px;margin:auto;font-size:18px">
+      <tr>
+        <td style="padding:4%;">
+          <%= yield %>
+        </td>
+      </tr>
+      <% if @user %>
+        <tr>
+          <td style="padding:3% 12% 2% 4%;font-size:18px;line-height:22px;color:#7d7d7d">
+            <%= signed_up_with(@user) %>
```

Comment:
Be sure to view this file with whitespace changes hidden -- most of this is me indenting the file because the elements were not indented correctly. The only _content_ change here is on line 12.